### ISSUE TITLE
Sync with shellinabox/shellinabox

### ISSUE THREAD

**Issue by marado** [uudruid74/shellinabox_fork#9]
As stated in README, and not done since Jan 2015, this fork should sync with https://github.com/shellinabox/shellinabox , or, alternatively, the changes made here should be transformed into pull requests to that project.

As they are right now, these two projects have different changes to the original shellinabox, which is a suboptimal scenario for users of one or the other.

**Comment by uudruid74** [comment]
Yeah, and I wanted to ditch the pre-processor cruft to simply things, and only modified the JS file, dumping the JSPP file, but @KLuka has maintained the JSPP file.  So merging will be a pain.  I'm going to look into it, but neither branch is much maintained anymore.

**Comment by uudruid74** [comment]
The sync is done.  I'm going to use uudruid74/shellinabox as a "stable" tree to match upstream and this tree will be used for more radical changes should I ever get the time for them.  As of right now, upstream (meaning shellinabox/shellinabox) had all the changes I did and did a better job with full 256 color support.  Only thing I'm ahead on is the gesture hack and a redesigned menu.
